x/cdp/simulation/operations/msgs.go

```diff
@@ -114,10 +122,23 @@ func SimulateMsgCdp(ak auth.AccountKeeper, k cdp.Keeper, pfk pricefeed.Keeper) s
 		if shouldDraw(r) {
 			collateralShifted := ShiftDec(sdk.NewDecFromInt(existingCDP.Collateral.AmountOf(randCollateralParam.Denom)), randCollateralParam.ConversionFactor.Neg())
 			collateralValue := collateralShifted.Mul(priceShifted)
+			// given the current collateral value, calculate how much debt we could add while maintaining a valid liquidation ratio
 			debt := (existingCDP.Principal.Add(existingCDP.AccumulatedFees)).AmountOf(randDebtParam.Denom)
 			maxTotalDebt := collateralValue.Quo(randCollateralParam.LiquidationRatio)
 			maxDebt := maxTotalDebt.Sub(sdk.NewDecFromInt(debt)).TruncateInt().Sub(sdk.OneInt())
```

Comment:
Why is `.Sub(sdk.OneInt())` applied here?

Reply:
@karzak 7 lines are shown here but I only selected 1. Any ideas on https://github.com/Kava-Labs/kava/issues/400?

Reply:
No ideas I'm afraid, weird issue. Every issue I see on that topic is people who can't select multiple lines, not people who can ONLY select multiple lines. Bizarre.

Reply:
As for the question, I think I was originally using `RoundInt` which could have drawn too much debt if it rounded up. Removed since `TruncateInt` already prevents this from happening.

Reply:
Yeah had the same results when searching earlier, super strange.